zebra/zebra_nb_config.c

```diff
@@ -881,12 +881,15 @@ int lib_interface_zebra_ip_addrs_destroy(struct nb_cb_destroy_args *args)
 	struct prefix prefix;
 	struct connected *ifc;
 
-	ifp = nb_running_get_entry(args->dnode, NULL, true);
```

Comment:
`destroy()` callback again

Reply:
Let's keep it here, please, to remove shared `nb_running_get_entry` in validate and apply stages. I think we can completely remove the dependency on the operational state from the validation stage here, so it's better to remove shared code.

Reply:
Fair enough.